app/managers.py

```diff
@@ -0,0 +1,36 @@
+import sqlite3
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._conection = sqlite3.connect(r"D:\Games\cinema.db")
```

Comment:
Ensure that the database path `D:\Games\cinema.db` is correct and accessible from your environment. If this path is incorrect, the connection will fail.